airbyte-webapp/build.gradle

```diff
@@ -23,6 +23,7 @@ node {
 npm_run_build {
     inputs.files commonConfigs
     inputs.file '.eslintrc'
+    inputs.file '.stylelintrc'
```

Comment:
Sorry, one last thing :sweat_smile: But with running stylelint as a separate task, we should move this as an input to the `stylelint` task below. The `npm run build` task would have no different output if this file changes.

Reply:
Fixed I think. You know gradle better than I :)